homeassistant/components/varta_storage/translations/de.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Don't add foreign languages. Additional translations can be provided via our Translation platform once this PR is merged.

Remove this file for now.

Reply:
okay